### PR TITLE
Product list Page

### PR DESCRIPTION
**Pull request by foolhardy21**
This PR has:
- Product List Page.

Feel free to review. Thanks.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    "aside main"
+    "aside main";
+    grid-template-columns: 15rem 1fr;
+    row-gap: 1rem;
+}
+
+#page-header {
+    grid-area: header;
+    background : var(--c-black);
+}
```
Comment (partha8) [inline style.css]: avoid using id selectors in css, https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4 you can refer this link.

**Review by partha8 — APPROVED** [review]
Looks good, just change id selector to class

**Review by vishal804 — APPROVED** [review]

**Comment by vishal804** [comment]
Everything Else looks Good

File: style.css (hunk 1 above)
Comment (foolhardy21) [reply style.css]: thanks for sharing @partha8 . i have made the changes

File: product-list.html [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+                    <input type='radio' class='margin-right-md' name="rating-group" id='3-atleast' />
+                    <label for='4-atleast' class="text-capital">atleast 3 stars</label>
+                </div>
+                <div class='flex-vertical-center margin-btm-sm'>
+                    <input type='radio' class='margin-right-md' id='2-atleast' name="rating-group"/>
+                    <label for='4-atleast' class="text-capital">atleast 2 stars</label>
+                </div>
+                <div class='flex-vertical-center margin-btm-sm'>
+                    <input type='radio' class='margin-right-md' name="rating-group" id='1-atleast' />
+                    <label for='4-atleast' class="text-capital">atleast 1 star</label>
```
Comment (anik31) [inline product-list.html]: Change the for attribute in labels to their corresponding inputs. Also try to keep the attributes in same order in all the inputs.

Comment (foolhardy21) [reply product-list.html]: thanks for pointing out @anik31 . i have updated the code.

**Pull request merged**